### DISCUSSION TITLE
Iraqi Kurdistan Guide - Discussions/Ideas

### DISCUSSION THREAD

**Discussion by TravelTimN** [TravelTimN/iraqikurdistanguide#57] [Ideas]
## 👋 Choni Bashi! 🏴󠁩󠁲󠀱󠀶󠁿

  Use this Discussions Board for:
  * Ask questions you’re confused about.
  * Share ideas you have for future implementations.

  Some tips:

  ➡️ You can **convert issues to discussions**, and have further enhanced discussions about them here.

**Comment by TravelTimN** [comment]
@10xOXR 
What are your thoughts about the **Booking** page for layout?
I can't put the buttons/select elements in-line or next to the `Month Year` above the calendar, so currently they sit above the the entire calendar.
Any thoughts or suggestions on how the look/layout should be?

**Reply by 10xOXR** [reply]
Other, offline discussions have since taken place and the current format of the Booking page now seems to work well for its intended purpose. It'll no doubt need to be tweaked further after client feedback but is good as-is.

**Reply by TravelTimN** [reply]
Thanks for your help on the `Q()` queries on the bookings!
- Current month only (June)
- Current month extending into the next month also (June into July)
- Previous month extending into the current month (May into June)
- Current month extending into the next year (Dec 2022 into Jan 2023)
- Previous year extending into current month (Dec 2022 into Jan 2023)
- Non repeats into the wrong year (2022 don't display on 2023, etc)
